spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/actions/DeleteOrphanFilesSparkAction.java

```diff
@@ -103,11 +102,14 @@ public class DeleteOrphanFilesSparkAction extends BaseSparkAction<DeleteOrphanFi
     implements DeleteOrphanFiles {
 
   private static final Logger LOG = LoggerFactory.getLogger(DeleteOrphanFilesSparkAction.class);
-  private static final Splitter COMMA = Splitter.on(",");
   private static final Map<String, String> EQUAL_SCHEMES_DEFAULT = ImmutableMap.of("s3n,s3a", "s3");
```

Comment:
Sorry I'm late to other review and might have missed the discussion, but the format of equalSchemes and equalAuthorities looks a bit confusing for a user to set.  Why is it a map of (list -> value) and not the other way around?  Or what is the differentiation between key/value , I imagine if everything of key/value is equal, it should be a list of lists?

Reply:
We wanted to map all the different schema./authorities to a single scheme/authority.
It seemed better, in terms of configuring, to have this map of list -> value.
For example ((s3n, s3a ) -> s3), in this case s3n and s3a will be mapped to s3.

Reply:
Thanks, is there any significance to s3?  Example, in this configuration are we actually expecting a value of s3://..., and treating it as equivalent to s3a:// and s3n://?   I guess my question is, if not, why not just have a list of (s3, s3a, s3n)?

Reply:
Is there any significance to s3:// , example in this use case are we expecting any s3:// files,  or just s3a:// , s3n://?  I would think we can just do a List(s3a://, s3n://), and internally generate some alias.  Or if we are expecting, then List(s3a://, s3n://, s3://)

Reply:
I was wondering this as well recently when looking at the format of equal authorities.

For the common case of (s3a, s3n) -> (s3), s3 is still an expected value correct?

I like the idea quite a lot and it's fairly clean, but for working with it it seems like Map(s3 -> s3a,s3n,s3) or set of all values would be easier to use programmatically.

But it's easy enough to map the values to what we need.

To clarify: (s3a,s3n) -> (s3) signifies that all 3 values should be used but we should write out s3 whenever possible, correct?

Sorry I missed the discussion on this.

Reply:
We thought about using a list but what if the table is persisted across multiple file systems and we have to map different schemes/authorities into different aliases?

I read this argument as consider `s3n` and `s3a` equal  to `s3`, which seems reasonable, but we can change this for sure. It hasn't been released. The current representation also fits into the way it is used in the normalization function right now.

@szehon-ho @kbendick, what would you change this to?